blog/20221108-simplex-chat-v4.2-security-audit-new-website.md

```diff
@@ -82,3 +84,96 @@ Beta features (enable Developer tools to try them):
+
+You can join these groups either by opening these links in the app or by opening them in desktop browser and scanning QR code.
+
+Let me know if you'd like to add some other countries to the list, and join via the apps to share what's going on and ask any questions!
+
+### Auto-accept contact requests
+
+When somebody connects to you via your long-term address you have to manually accept a connection request (it shows in blue color in the list of chats). The feature that we added in this release allows to configure the app to accept contact requests automatically, and also choose whether this contact should receive your main profile or a random incognito profile (independent of the current app setting), and add an optional auto-reply message.
+
+This feature is useful if you publish your address on your webpage or social profile, and do not want to screen people who want to connect to you. You may want to send a standard welcome message, for example, if it is an online store, and you need to share any information with everybody who contacts you.
+
+Our @simplex account that you connect to when you choose "Connect to developers" in the app used this feature for a long time, and now it is available to mobile app users.
+
+### Some small things
+
+1. Change group member role is a very basic feature, but it was only added in this release.
+
+2. You can now mark the whole conversation as unread, for example if you accidentally marked all messages as read and you want to review it later.
+
+3. You now can send stickers and GIFs from Android keyboards, and, finally, the bug with backspace button is resolved as well.
+
+### Change your delivery address (BETA)
+
+To manually switch any of your contacts (or a group member to a new server address) enable Developer tools and choose "Change receiving address" on the contact page. As long as they run a new version of the app and online, the switch should only take a few seconds.
+
+That is a major improvement of metadata privacy of SimpleX protocols, because previously, while we didn't have user identifiers, the pairwise identifiers of messaging queues used to deliver messages were used for as long as you the contact existed. Now these identifiers are temporary, and in a near future we will be adding automatic rotation of these delivery addresses.
```

Comment:
```suggestion
That is a major improvement of metadata privacy of SimpleX protocols, because previously, while we didn't have user identifiers, the pairwise identifiers of messaging queues used to deliver messages were used for as long as the contact existed. Now these identifiers are temporary, and in a near future we will be adding automatic rotation of these delivery addresses.
```